appjson/healthcheck.go

```diff
@@ -129,6 +138,31 @@ func (h Healthcheck) Execute(container types.ContainerJSON, ctx HealthcheckConte
 	return h.executeUptimeCheck(container)
 }
 
+func (h Healthcheck) HandleFailure(errors []error) []error {
```

Comment:
I think we should only need to return a single error ever from this func.

Reply:
you're right, changed it